inscriptions/ISic003639.xml

```diff
@@ -186,33 +191,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR113464" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
                 <ab>
-                    <lb n="1"/>
-                    (
-                    <expan>
-                        <ex>luna crescente</ex>
-                    </expan>
-                    )
-                    <lb n="2"/>
-                    <expan>
-                        <abbr>D</abbr>
-                        <ex>iis</ex>
-                    </expan>
-                    <expan>
-                        <abbr>M</abbr>
-                        <ex>anibus</ex>
-                    </expan>
-                    .<lb n="3"/>Elvius Titus<lb n="4"/>vixit annos<lb n="5"/>LV, cives be<lb n="6" break="no"/>ne
-                    merenti<lb n="7"/>fecerunt.
+                    <lb n="1"/> <g type="crescentmoon"/> <!-- I cannot find a unicode symbol for this, or any ref in the database -->
+                    <lb n="2"/> <g ref="leaf">❧</g> <expan><abbr>D</abbr><ex>is</ex></expan> <expan><abbr>M</abbr><ex>anibus</ex></expan> <g type="leaf">❧</g>
```

Comment:
ref="#ivy_leaf" - and always use the same unicode symbol (consistently the ❦ one).
we will add crescent moon. I suggest for now you put: `<g ref="#moon_crescent"/>`